backend/src/main/kotlin/blue/mild/covid/vaxx/dao/Patient.kt

```diff
@@ -8,7 +8,7 @@ import java.time.Instant
 object Patient : Table("patients") {
     val id = varchar("id", 36)
     val created: Column<Instant> = timestamp("created")
-    val updated: Column<Instant?> = timestamp("updated").nullable()
+    val updated: Column<Instant> = timestamp("updated")
```

Comment:
Bacha, v migraci je update nullable. Ale tak to kdyztak zmenme na NOT NULL.

Reply:
fixed v #78